internal/cmd/auth/login.go

```diff
@@ -99,6 +99,12 @@ func writeDefaultOrganization(ctx context.Context, accessToken, authURL string)
 		return err
 	}
 
+	// If the configuration directory doesn't exist, we create it.
+	err = config.CreateConfigDir()
+	if err != nil {
+		return err
+	}
```

Comment:
What about moving this into `WriteDefault()` instead? Where we do the actual file write?

Reply:
Good call, thanks!